content/en/account_management/org_settings/ip_allowlist.md

```diff
@@ -0,0 +1,88 @@
+---
+title: IP Allowlist
+kind: documentation
+---
+
+{{< callout url="/help/" header="Get Started with IP Allowlist" >}}
+The IP allowlist feature is available for enterprise-level customers only. Request access by contacting support.
+{{< /callout >}}
+
+## Overview
+
+The IP allowlist controls which networks can be used to access your data in Datadog. By limiting allowed networks, you can protect your resources from data exfiltration and insider threats.
+
+When the IP allowlist is enabled, only IP addresses or CIDR ranges in the allowlist can access the Datadog API and UI. 
+
+### Blocked and allowed resources
+
+If a user's IP is not contained in the IP allowlist, they are effectively blocked from accessing and using:
+
+- Datadog’s web UI
+- Datadog’s public [API][1], including both documented and unpublished endpoints
+- Datadog’s mobile apps (iOS, Android)
+- Third-party applications that access Datadog through OAuth
+
+The IP allowlist feature does not block access to the following:
+- Data ingest endpoints to which the Agent sends data, such as metrics, traces, and logs
+- The [Validate API key][2] endpoint, which is often used by data ingestion
+- [Public dashboards][3]
+
+### Functionality
+
+Only users with the **Org Management** permission can configure the IP allowlist.
+
+With the IP allowlist API or UI, you can:
+- Check the status of the IP allowlist. Whether the IP allowlist is on or off determines whether your organization is restricting requests by IP address allowlist membership.
+- Turn the IP allowlist on and off.
+- Show the IP addresses (as CIDR ranges) that are covered by your IP allowlist.
+- Add IP addresses (IPv4 or IPv6) or CIDR ranges to the IP allowlist with an optional note.
+- Edit the note for an IP address already in the IP allowlist.
+- Delete a single entry from the IP allowlist.
+- Replace the whole IP allowlist with new entries (only available through the API).
+
+## Managing the IP allowlist in the UI
```

Comment:
Could we consider including screenshots/gif (in addition to written instructions) on how customers can manage IP Allowlist? I think it would be a more user friendly experience

Reply:
Hi! I agree that screenshots make the page much more user-friendly. We do limit screenshots because the UI changes so often that they create a maintenance problem for the docs team.

The bigger issue here is that I do not have access to an org where I can take a screenshot. I don't have org_management in org2. Do you know of a screenshot I can use or who could take one for me?

Reply:
I just invited you to an org in us1 called "IP Allowlist Test" which we use for allowlist testing. We could probably populate it with some representative/fake data.

Note: it can be a pain because several of us use it for testing so sometimes we get locked out. Please disable the allowlist when you aren't actively testing.

Reply:
@urseberry I took 2 screenshots to add to the docs!

List of IPs, belongs somewhere at the top: https://a.cl.ly/xQuROLXE 
Modal to add IP, belongs in the add IPs section: https://a.cl.ly/eDurnPYr